sdk/go/common/apitype/core.go

```diff
@@ -329,6 +329,10 @@ type ResourceV3 struct {
 	// If set, the providers Delete method will not be called for this resource
 	// if specified resource is being deleted as well.
 	DeletedWith resource.URN `json:"deletedWith,omitempty" yaml:"deletedWith,omitempty"`
+	// CreatedAt tracks when the remote resource was created in the provider by pulumi. Checkpoints prior to early 2023 do not include this.
+	CreatedAt *time.Time `json:"createdAt,omitempty" yaml:"created,omitempty"`
+	// UpdatedAt tracks when the resource state was last modified to reconcile a difference between a provider and the state file. Checkpoints prior to early 2023 do not include this.
+	UpdatedAt *time.Time `json:"updatedAt,omitempty" yaml:"modified,omitempty"`
```

Comment:
I thought we were going with "ModifiedAt"?

Reply:
Everyone seemed to have an opinion so I took a Poll and I linked it in our internal discussion.